databox/api/migrations/Version20250318094304.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250318094304 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Stories: bilateral OneToOne asset <-> collection';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('DROP INDEX idx_2af5a5c717dfdc8');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_2AF5A5C717DFDC8 ON asset (story_collection_id)');
+        $this->addSql('ALTER TABLE collection DROP story_asset_id');
```

Comment:
There is no column `story_asset_id`.
I think you've messed up with your local schema. You need to revert from master, update the schema, then go back to your branch and generate a fresh migration.